data/tlg0062/tlg054/tlg0062.tlg054.perseus-eng3.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://epidoc.stoa.org/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+    <teiHeader xml:lang="eng">
+<fileDesc>
+<titleStmt>
+<title xml:lang="eng">The Dipsads</title>
+<title xml:lang="grc">Περὶ τῶν διψάδων</title>
+<author>Lucian of Samosata</author>
+<editor role="translator">Kenneth Kilburn</editor>
+<sponsor>Perseus Project, Tufts University</sponsor>
+<principal>Gregory Crane</principal>
+<respStmt>
+<resp>Prepared under the supervision of</resp>
+<name>Gregory Crane</name>
+<name>Lisa Cerrato</name>
+
+</respStmt>
+<funder n="org:NEH">The National Endowment for the Humanities</funder>
+</titleStmt>
+<extent/>
+<publicationStmt>
+<publisher>Trustees of Tufts University</publisher>
+<pubPlace>Medford, MA</pubPlace>
+<authority>Perseus Project</authority>
+    <date type="release" when="2023-06-23">Perseus 5.0</date>
+    <idno type="filename">tlg0062.tlg053.perseus-grc3.xml</idno>
+    <availability>
+        <licence target="https://creativecommons.org/licenses/by-sa/4.0/">Available under a Creative Commons Attribution-ShareAlike 4.0 International License</licence>
+    </availability>
+</publicationStmt>
+
+    <sourceDesc>
+        <listBibl>
+            <biblStruct>
+                <monogr>
+                    <author>Lucian of Samosata</author>
+                    <title xml:lang="eng">Lucian with an English Translation</title>
```

Comment:
We don't to mark English titles as English since we just do that for the whole header. We also don't tend to include phrases like "an English translation" but that's just semantics.

Reply:
1. I actually went through and added the xml:lang="eng" last night. Often, Latin and English short titles look identical and should be distinguished.
2. "Lucian" is just too damned short. Give people as much info as we can when they hunt through library catalogs.